services/cases-api/lambdas/updateCase.js

```diff
@@ -9,17 +13,44 @@ import { decodeToken } from '../../../libs/token';
 import { objectWithoutProperties } from '../../../libs/objects';
 import { getStatusByType } from '../../../libs/caseStatuses';
 
+import { getCaseWhereUserIsApplicant } from '../helpers/dynamo';
+import { updateCaseValidationSchema } from '../helpers/schema';
+
 export async function main(event) {
   const requestJsonBody = JSON.parse(event.body);
   const { id } = event.pathParameters;
+  const { personalNumber } = decodeToken(event);
 
   if (!id) {
-    return response.failure(new BadRequestError('missing [id] in query path'));
+    return response.failure(new BadRequestError('Missing required path parameter "id"'));
+  }
+
+  const { error, validatedEventBody } = updateCaseValidationSchema.validate(requestJsonBody, {
+    abortEarly: false,
+  });
+  if (error) {
+    response.failure(new BadRequestError(error.message.replace(/"/g, "'")));
+  }
+
+  const {
+    provider,
+    details,
+    currentFormId,
+    currentPosition,
+    answers,
+    signature,
+  } = validatedEventBody;
```

Comment:
I couldn't run `sls deploy` without running `prettier` on this part first, as it throws an error here. 

`prettier` should be always be used regardless anyway. Make sure to setup it to run on save for your IDE (VS Code has a prettier extension, then set `prettier` as your default formatter).

Reply:
Had problems with prettier, seems like it was a mismatch between how serverless bundle runs. Turning prettier of and letting eslint handle the formatting fixed this issue for me. But happy for other suggestions, since not having prettier enabled in vscode is not ideal :)

Reply:
My suggestion would be to never let ESLint handle formatting (only linting), and let Prettier handle all formatting, and then apply the same eslintconfig and prettierrc across all js/ts projects in the future. But for now, if it deploys then it's fine! 🙂 Others might have to verify (or perhaps we can disable/ignore prettier errors on deploy; not sure how it is setup).